js/packages/teams-ai/package.json

```diff
@@ -26,7 +26,7 @@
     "types": "./lib/index.d.ts",
     "peerDependencies": {
         "botbuilder": "^4.22.1",
-        "openai": "4.28.4"
+        "openai": "^4.41.0"
```

Comment:
Could you please check that all of our samples are now using v 4.41.0 as well? As a peer dependency we've added them to all of the samples.

Will there be a sample for streaming?

Reply:
sure and I assumed the streaming sample would happen once the Teams Client actually supports streaming.